addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -41,14 +43,14 @@ def setUpClass(cls):
     @users('user_public')
     def test_access_public(self):
         # Read public channel -> ok
-        self.env['discuss.channel'].browse(self.public_channel.id).read()
```

Comment:
note: read() is not great because some related fields added by other modules are not accessible, which made the test not runnable locally under certain module setup

the goal was just to test accessing any field